plugins/compress/configuration.cc

```diff
@@ -188,11 +188,7 @@ HostConfiguration::is_content_type_compressible(const char *content_type, int co
   return is_match;
 }
 
-int
-isCommaOrSpace(int ch)
-{
-  return (ch == ',') or isspace(ch);
-}
+constexpr auto isCommaOrSpace = [](int ch) -> int { return (ch == ',') || isspace(ch); };
 
```

Comment:
Is there an advantage to making this a lambda?  Maybe just leave this as it was, but make it a constexpr?